packages/legend-application/src/stores/AlertService.ts

```diff
@@ -29,11 +29,13 @@ export interface ActionAlertInfo {
   title?: string;
   message: string;
   prompt?: string;
+  content?: React.ReactElement;
```

Comment:
we can change `title?: React.ReactNode` right?

Reply:
You mean changing label to be `React.ReactNode`? Title belongs to modal header I believe. I was thinking if we change label though, almost every existing label will have to be text changed to `label: <>text here</>` which seems less readable because in almost every case label now and later may be string.